_rules/motion-actuation-bf0de1.md

```diff
@@ -0,0 +1,737 @@
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] that has [functionality][] that can be operated by device motion or user motion.
+
+## Expectation (1)
+
+For each test target, device motion or user motion [can be disabled][].
+
+**Note:** the control for disabling should be visible and included in the accessibility tree.
+
+## Expectation (2)
+
+For each test target, device motion or user motion can also be operated by [user interface components][].
+
+## Assumptions
+
+- This test assumes that motion is not [essential][] for the [functionality][]
+
+## Accessibility Support
+
+- Device [orientation events][] may include sensitive data. Implementations must fire events only on [secure browsing contexts][]
+
+## Background
+
+- [Understanding Success Criterion 2.5.4: Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)
+- [G213: Provide conventional controls and an application setting for motion activated input](https://www.w3.org/WAI/WCAG21/Techniques/general/G213.html)
+- [DeviceOrientation Event Specification](https://www.w3.org/TR/orientation-event/)
+- [Detecting device orientation](https://www.w3.org/TR/orientation-event/#deviceorientation)
+- [DeviceMotionEvent](https://www.w3.org/TR/orientation-event/#devicemotion)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The [HTML document][] has [functionality][] that can be operated by device motion and can also be operated by [user interface components][] and [can be disabled][].
+
+```html
+<html>
+  <head>
+    <title>Passed Example 1</title>
+    <script>
+      function activateSlider() {
+        //slider behavior
+        var slider = document.getElementById("motionSlider");
+        var output = document.getElementById("demo");
+        output.innerHTML = slider.value;
+        slider.oninput = function () {
+          output.innerHTML = this.value;
+        }
+
+        document.getElementById('increase').addEventListener('click', function() {
+          slider.value++;
+          output.innerHTML = slider.value;
+        });
+
+        document.getElementById('decrease').addEventListener('click', function() {
+          slider.value--;
+          output.innerHTML = slider.value;
+        });
+
+        //slider motion detection
+        function handleOrientation(event) {
+
+          var z = document.getElementById("disable");
+          if (z.checked == false) {
+            var x = event.gamma;
+          } else {
+            x = 0;
+          }
+
+          if (x > 20) {
+            slider.value++;
+            output.innerHTML = slider.value;
+          } else if (x < -20) {
+            slider.value--;
+            output.innerHTML = slider.value;
+          }
+        }
+
+        window.addEventListener('deviceorientation', handleOrientation);
+      }
+    </script>
```

Comment:
I understand what you're saying, but most examples have subtle variations of the same code, and it's not reasonable to move everything to a separate asset test. Since most examples have their differences in the script code, i'll leave the code as it is. Nevertheless i did optimize most scripts code.